doc/deprecated.rst

```diff
@@ -130,3 +130,7 @@ Miscellaneous
 
 * As of Twig 1.x, ``Twig_Environment::clearTemplateCache()`` is deprecated and
   will be removed in 2.0.
+
+* As of Twig 1.x, ``Twig_Template::getEnvironment()`` and
+  ``Twig_TemplateInterface::getEnvironment()`` are deprecated and will be
+  removed in 2.0.
```

Comment:
The deprecation message says it's removed in 3.0.
`@deprecated since 1.20 (to be removed in 3.0)`